src/ngraph/ngraph_emitter.cc

```diff
@@ -1365,6 +1365,25 @@ void Emitter::CreateLayerOps() {
     return ngraph::builder::make_with_numpy_broadcast<ngraph::op::Add>(
         convolution, bias_reshape);
   };
+  ngraph_op_funcs_["Deconvolution"] =[this](const NodePtr& node) -> NgraphNodePtr {
```

Comment:
At one point we discussed implementing this logic as an nGraph Builder.  Is that no longer the plan?

Reply:
No, I think the other option would have gone into a builder.